pkgs/development/python-modules/triton/bin.nix

```diff
@@ -51,29 +51,8 @@ buildPythonPackage rec {
   # If this breaks, consider replacing with "${cuda_nvcc}/bin/ptxas"
   postFixup =
     ''
-      chmod +x "$out/${python.sitePackages}/triton/third_party/cuda/bin/ptxas"
-    ''
-    + (
-      let
-        # Bash was getting weird without linting,
-        # but basically upstream contains [cc, ..., "-lcuda", ...]
-        # and we replace it with [..., "-lcuda", "-L/run/opengl-driver/lib", "-L$stubs", ...]
-        old = [ "-lcuda" ];
-        new = [
-          "-lcuda"
-          "-L${addDriverRunpath.driverLink}"
-          "-L${cudaPackages.cuda_cudart}/lib/stubs/"
-        ];
-
-        quote = x: ''"${x}"'';
-        oldStr = lib.concatMapStringsSep ", " quote old;
-        newStr = lib.concatMapStringsSep ", " quote new;
-      in
-      ''
-        substituteInPlace $out/${python.sitePackages}/triton/common/build.py \
-          --replace '${oldStr}' '${newStr}'
-      ''
-    );
+      chmod +x "$out/${python.sitePackages}/triton/backends/nvidia/bin/ptxas"
```

Comment:
There was a refactor which seems to have changed the pathing for the various backends.